package.json

```diff
@@ -20,7 +21,7 @@
     "hardhat-abi-exporter": "^2.2.1",
     "hardhat-contract-sizer": "^2.0.3",
     "hardhat-deploy": "^0.7.10",
-    "hardhat-deploy-ethers": "*",
+    "hardhat-deploy-ethers": "^0.3.0-beta.13",
```

Comment:
Is there a more solid version? instead of beta?

Reply:
Beta 13 is the latest version, this is a dev dependency we use to deploy the contracts.

Reply:
yeah I get that, I guess my question is; since it is a `beta` version is there one that is not beta? if so, can we use it? Or is this a relatively new lib that does not have a solid version yet and we are kinda stuck using it as it is.

Reply:
https://github.com/wighawag/hardhat-deploy-ethers/tags alright ignore my previous message it looks like all versions are in `beta`